docs/_layouts/default.html

```diff
@@ -25,14 +25,21 @@
              {% for my_page in final_list %}
                 {% if my_page.nav_link and my_page.nav_link != blank and my_page.lang and my_page.lang != blank and my_page.url contains lang %}
                    {% if my_page.nav_level > current_level %}
-                       <ul>
+                       <ul class="list-nav">
                        {% assign current_level = my_page.nav_level %}
                    {% elsif my_page.nav_level < current_level %}
-                                                           </ul>
+                       </ul>
                          {% assign current_level = my_page.nav_level %}
                    {% endif %}
                    {% if my_page.url == page.url %}
                      <li><a class="page-link selected-page-link" href="{{ my_page.url | relative_url }}">{{ my_page.nav_link | escape }}</a></li>
+                     {% if my_page.subnav.size > 0 %}
```

Comment:
Will this work without the plug-in?  If not, it has to go.

Reply:
Related to changes to be completed in: https://jira.corp.adobe.com/browse/UMAPI-481. This does not break functionality.